packages/dds/tree/src/test/simple-tree/discrepancies.spec.ts

```diff
@@ -0,0 +1,909 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import assert from "node:assert";
+import {
+	LeafNodeStoredSchema,
+	MapNodeStoredSchema,
+	ObjectNodeStoredSchema,
+	storedEmptyFieldSchema,
+	ValueSchema,
+	type TreeFieldStoredSchema,
+	type TreeNodeSchemaIdentifier,
+	type TreeStoredSchema,
+} from "../../core/index.js";
+import {
+	defaultSchemaPolicy,
+	allowsRepoSuperset,
+	FieldKinds,
+	type FlexFieldKind,
+} from "../../feature-libraries/index.js";
+import { brand } from "../../util/index.js";
+import {
+	FieldKind,
+	normalizeFieldSchema,
+	SchemaFactoryAlpha,
+	toStoredSchema,
+	isRepoSuperset,
+	getAllowedContentDiscrepancies,
+	type AnnotatedAllowedType,
+	type TreeNodeSchema,
+} from "../../simple-tree/index.js";
+// eslint-disable-next-line import/no-internal-modules
+import { createFieldSchema } from "../../simple-tree/fieldSchema.js";
```

Comment:
avoid this reaching import just use static methods on public API

Reply:
Fine for now I guess, though be aware that non-string identifiers are intentionally not supported, and likely to break things so its quite possible what this test is doing will start asserting at some point.